docs/howtoguides/trusty_legacy_support.rst

```diff
@@ -0,0 +1,122 @@
+starting with Ubuntu 14.04 LTS <https://canonical.com/blog/canonical-expands-long-term-support-to-12-years-starting-with-ubuntu-14-04-lts>`_
+to read on about what the service provides and for links to contact Canonical to acquire that
+service.
+
+
+Make sure ``ua`` is up-to-date
+===============================
+
+All Trusty systems come with ``ua`` pre-installed through the
+``ubuntu-advantage-tools`` package. To make sure that you're running the latest
+version of ``ua``, run the following commands:
+
+.. code-block:: bash
+
+    sudo apt update && sudo apt install ubuntu-advantage-tools
+
+Note that on Trusty, Ubuntu Pro is referred to by its previous name: "Ubuntu Advantage" (or ``ua``).
+
+
+Check the status of the services
+================================
+
+After you have attached to a Pro subscription and
+updated the ``ubuntu-advantage-tools`` package, you can check which services
+are enabled by running the following command:
+
+.. code-block:: bash
+
+    sudo ua status
+
+This will show you which services are enabled or disabled on your machine,
+(output truncated for brevity). With the addition of legacy support, it will
+now show the esm-infra-legacy service on Trusty:
+
+.. code-block:: text
+
+    SERVICE          ENTITLED  STATUS    DESCRIPTION
+    esm-infra        yes       enabled   Expanded Security Maintenance for Infrastructure
+    esm-infra-legacy yes       disabled  Expanded Security Maintenance for Infrastructure on Legacy Instances
+    livepatch        yes       enabled   Canonical Livepatch service
+
+.. _enable_legacy:
+
+Enabling esm-infra-legacy
+==========================
+
+If you are entitled to the esm-infra-legacy service as shown above, you can enable it by running:
+
+.. code-block:: bash
+
+    sudo ua enable esm-infra-legacy
+
+
+What to do if esm-infra-legacy is not entitled
+===============================================
+
+If status is showing that you are not entitled to the service, you
+will need to buy access to the service. You can contact sales/support for help on that regard.
+
+Once you have bought support, you will need to run the following command
+on your machine to refresh the contract definitions:
+
+.. code-block:: bash
+
+    sudo ua refresh
+
+
+After refreshing the contract data, you can confirm that the service is now entitled by running:
+
+
+.. code-block:: bash
+
+    sudo ua status
+
+Now that you are entitled and can go refer to this :ref:`section <enable_legacy>` to enable
+the service.
+
+
+Trusty caveats
+===============
+
+There are some known caveats for the Trusty version of `ubuntu-advantage-tools`:
+
+* **Disabling esm-infra also disables esm-infra-legacy**: If you disable esm-infra that will due
+  to internal dependencies also disable esm-infra-legacy. Even though further updates will only
+  come in via esm-infra-legacy it is recommended to keep both enable. If desired to disable
+  esm-infra-legacy, that is ok and can be done without consequences for esm-infra.
```

Comment:
```suggestion
Disabling ``esm-infra`` also disables ``esm-infra-legacy``
----------------------------------------------------------

If you disable ``esm-infra``, this will (due to internal dependencies) also disable
``esm-infra-legacy``. Although updates will **only** be applied via ``esm-infra-legacy``,
we recommend keeping both services enabled. This is not true in reverse: if you want
to disable ``esm-infra-legacy``, doing so will not disable ``esm-infra``.
```